pymc/tests/test_distributions_timeseries.py

```diff
@@ -223,27 +207,27 @@ def test_gaussian_random_walk_init_dist_logp(self, init):
             pm.logp(init, 0).eval() + scipy.stats.norm.logpdf(0),
         )
 
-    @pytest.mark.parametrize(
-        "mu, sigma, init_dist, steps, size, expected",
-        [
-            (0, 1, Normal.dist(1), 10, None, np.ones((11,))),
-            (1, 1, Normal.dist(0), 10, (2,), np.full((2, 11), np.arange(11))),
-            (1, 1, Normal.dist([0, 1]), 10, None, np.vstack((np.arange(11), np.arange(11) + 1))),
-            (0, [1, 1], Normal.dist(0), 10, None, np.zeros((2, 11))),
-            (
-                [1, -1],
-                1,
-                Normal.dist(0),
-                10,
-                (4, 2),
-                np.full((4, 2, 11), np.vstack((np.arange(11), -np.arange(11)))),
-            ),
-        ],
-    )
-    def test_moment(self, mu, sigma, init_dist, steps, size, expected):
-        with Model() as model:
-            GaussianRandomWalk("x", mu=mu, sigma=sigma, init_dist=init_dist, steps=steps, size=size)
-        assert_moment_is_expected(model, expected)
+    # @pytest.mark.parametrize(
```

Comment:
Why does this test fail now?

Reply:
I forgot to uncomment it for the commit. The `init_dist` had to be fetched by going up the graph a bit and now the tests are passing :)